packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -128,7 +128,17 @@ class HotRunner extends ResidentRunner {
       printError('Error initializing DevFS: $error');
       return 3;
     }
+    Stopwatch initialUpdateDevFSsTimer;
+    if (benchmarkMode) {
+      initialUpdateDevFSsTimer = new Stopwatch();
+      initialUpdateDevFSsTimer.start();
+    }
     final bool devfsResult = await _updateDevFS();
+    if (benchmarkMode) {
+      initialUpdateDevFSsTimer.stop();
```

Comment:
nit: you don't need to stop the `Stopwatch` if you do not plan to resume it later.